### PR TITLE
[TeamQuest] Add sidebar view and sidebar template

### PR DESCRIPTION
**Pull request by razvan3895**

### CODE REVIEW

File: wouso/games/teamquest/views.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        progress = 0
+        status = None
+
+    return render_to_string('teamquest/sidebar.html',
+            {'quest': quest, 'tquser': quest_user,
+             'group': quest_user.group, 'status':status,
+             'progress': progress, 'id': 'teamquest'
+             })
+
+register_sidebar_block('teamquest', sidebar_widget)
```
Comment (iulianR) [inline wouso/games/teamquest/views.py]: newline :octocat:

File: wouso/games/teamquest/templates/teamquest/sidebar.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+{% block title %}{% trans 'Team Quest' %}{% endblock %}
+
+{% block content %}
+
+{% if quest and group %}
+    <p> {{ quest.title }} </p>
+    {% if status %}
+        {% if status.progress < quest.total_points %}
+            <div class="progress-bar">
+                    <div class="progress" style="width: {{ progress|floatformat:0 }}%"></div>
```
Comment (iulianR) [inline wouso/games/teamquest/templates/teamquest/sidebar.html]: Keep indentation consistent. (4 spaces)

File: wouso/games/teamquest/templates/teamquest/sidebar.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            <div class="progress-bar">
+                <div class="progress" style="width: {{ progress|floatformat:0 }}%"></div>
+            </div>
+            <p><a href="">{% trans 'Play quest' %}</a></p>
+        {% endif %}
+    {% else %}
+        <p><a href="">{% trans 'Start quest' %} </a></p>
+    {% endif %}
+{% elif quest and not group %}
+    <p> {% trans 'There is an active quest, but you need a team to play' %} </p>
```
Comment (razvand) [inline wouso/games/teamquest/templates/teamquest/sidebar.html]: Add dot (`.`) at the end of each report sentence. My suggestion is that all report sentences should end in dot. This is not the case for `Play quest` for example, because that is a command/action not a report.

File: wouso/games/teamquest/templates/teamquest/sidebar.html [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+{% else %}
+    <p> {% trans 'There is no active quest' %} </p>
+{% endif %}
+
+
+<p><a href="">{% trans 'Team hub' %}</a></p>
```
Comment (razvand) [inline wouso/games/teamquest/templates/teamquest/sidebar.html]: Should be `Team Hub` with capital `H`.

File: wouso/games/teamquest/tests.py [hunk 5]
```diff
@@ -158,25 +183,7 @@ def setUp(self):
         self.group = TeamQuestGroup.create(group_owner=self.owner, name='_test_group')
         category = Category.add('quest')
```
Comment (razvand) [inline wouso/games/teamquest/tests.py]: Why is this here? It's already part of the `_create_quest` method.
